packages/geocoder/src/geocoders/abstract-geocoder.ts

```diff
@@ -18,17 +18,21 @@ type Boundary = {
 };
 
 type GeocoderAPI = {
-  autocomplete: (query: AutocompleteQuery) => Promise<Array<JSON>>;
-  search: (query: SearchQuery) => Promise<Array<JSON>>;
-  reverse: (query: ReverseQuery) => Promise<Array<JSON>>;
+  autocomplete: (query: AutocompleteQuery) => Promise<unknown>;
+  reverse: (query: ReverseQuery) => Promise<unknown>;
+  search: (query: SearchQuery) => Promise<unknown>;
 };
 
+// These settings get added to the request to the geocoder
 export type GeocoderConfig = {
   apiKey?: string;
   baseUrl?: string;
   boundary?: Boundary;
   focusPoint?: LonLatInput;
   options?: RequestInit;
+  sources?: string;
+  size?: number;
+  reverseUseFeatureCollection?: boolean;
 };
 
 export type ReverseQuery = {
```

Comment:
Any thoughts @daniel-heppner-ibigroup @miles-grant-ibigroup about creating an interface with some of the fields common to all query types below? (this is not blocking)

Reply:
Could be good! Could put it in the types package. But agree not needed for this PR